frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap

```diff
@@ -61,11 +61,11 @@ Array [
 ]
 `;
 
-exports[`<ActionMessage /> renders Stake 1`] = `"Staked: <0.00001 Ⓝ ed25519:BgXFiJS..."`;
+exports[`<ActionMessage /> renders Stake 1`] = `"Staked: 0 Ⓝ ed25519:BgXFiJS..."`;
 
 exports[`<ActionMessage /> renders Transfer 1`] = `
 Array [
-  "Transferred <0.00001 Ⓝ to ",
+  "Transferred 0 Ⓝ to ",
```

Comment:
We should fix `formatNEAR` to actually reflect that the amount is not 0, but rather very small.

@janedegtiareva Can nearlib format small yoctoNEAR amounts in a human-friendly format, e.g. `1 yoctoN`? We used to have `<0.00001 N` here. Do you think it is reasonable?

@icerove @kcole16 It seems that to make the information non-confusing and useful, we should present both `<0.00001 N (1 yoctoNEAR)` (we may want to show the exact amount, e.g. "1 yoctoNEAR", as an on-hover popup)

Reply:
@icerove I still don't see this resolved. "Transferred 0 N" is not useful.

Reply:
Well, this same problem is going to be in Wallet.

@vgrichina @janedegtiareva How do you display small NEAR amounts in Wallet? It makes sense to have 1 yoctoNEAR rendered as something more human-friendly than 0 N.

Reply:
@frol wallet still displays `<0.00001 N`. 

https://github.com/nearprotocol/near-wallet/blob/41cb65246134308dd553b532dfb314b45b38b65c/src/components/common/Balance.js#L26